pipeline.yml

```diff
@@ -9,13 +9,21 @@ jobs:
     - get: common
       resource: common-ops-staging
       trigger: true
+    - get: terraform-state
+      resource: state-file-tooling
+      trigger: true
+    - get: pipeline-tasks
     - get: concourse-stemcell
       trigger: true
     - get: concourse-release
       trigger: true
     - get: garden-runc-release
       trigger: false
     - get: master-bosh-root-cert
+  - task: terraform-state-to-yaml
```

Comment:
WDYT about moving this to the terraform pipeline itself so we do it once, rather than putting this in all our pipelines?